iphone/Classes/TiUIButton.m

```diff
@@ -412,26 +412,9 @@ -(void)setVerticalAlign_:(id)align
 -(void)setTextAlign_:(id)align
 {
     button = [self button];
-    NSTextAlignment alignment = NSTextAlignmentNatural;
```

Comment:
Agree. I'd rather change [this constant](https://github.com/appcelerator/titanium_mobile/blob/master/iphone/Classes/TiUtils.m#L1354) to `NSTextAlignmentNatural`  though. They look the same, but `NSTextAlignmentNatural` is the native way to go and default since iOS 6.